README.md

```diff
@@ -88,34 +105,44 @@ yum install -y epel-release
 yum install -y readline-devel leveldb-devel libffi-devel gcc-c++ redhat-rpm-config gcc python-devel openssl-devel
 ```
 
-### For all of these, make sure that the `Chains` directory in your project has the proper write permissions
+#### Windows
 
-##### Windows
+Help needed. Installing the Python package plyvel seems to require C++ compiler
+support tied to Visual Studio and libraries.
 
-Not sure. Installing the Python package plyvel seems to require C++ compiler support tied to Visual Studio and libraries.
+### Python 3.5
 
--------------------
+neo-python is currently only compatible with **Python 3.5** (due to its `byteplay3` dependency).
+
+On *nix systems, install Python 3.5 via your package manager, or download an installation package
+from the [official homepage](https://www.python.org/downloads/release/python-354).
 
-### Virtual Environment
+##### Virtual Environment
```

Comment:
wrong way -- i think it should be level 3 as it was, the last commit had it at level 4, now it's at level 5. just make it as it was with `### Virtual ...`